src/less/awful/ssl.clj

```diff
@@ -80,16 +80,14 @@
 (defn key-store
   "Makes a keystore from a PKCS8 private key file, a public cert file, and the
   signing CA certificate."
-  [key-file cert-file ca-cert-file]
+  [key-file cert-file]
```

Comment:
I don't pretend to understand SSL but I was under the impression that only the trust-store needs the CA cert. The key-store needs the real (signed) cert and the associated private key.

In my foolings with nginx and client certs in browsers, the server needed the CA cert that was used to sign the client-side cert in order to trust the client, but the client only needed the certificate and key. I believe the CA cert might also be used as a hint to clients telling them which cert to use to communicate with the server.